legacy/ui/legacy/src/main/java/com/fsck/k9/ui/messagelist/MessageListAdapter.kt

```diff
@@ -161,6 +174,9 @@ class MessageListAdapter internal constructor(
         listItemListener.onToggleMessageSelection(messageListItem)
     }
 
+    private val isInAppNotificationEnabled: Boolean
```

Comment:
In one of the upcoming changes I added a shorthand change to the `FeatureFlagResult` to check for `isEnabled()`, see #9797